### ISSUE TITLE
Lane change via TraCI rejected with default sublane lane change mode on roads with more than two lanes

### ISSUE THREAD

**Issue by mpfist-no** [eclipse/sumo#12810]
With the default lane change mode, Sumo does not apply certain lane changes requested from TraCI.

I noticed that this seems to occur only on roads with more than two lanes. A lane change to the third lane from the right is the one that gets rejected. Lane changes to the right seem to be accepted always (as expected).

The lane change is executed correctly if the 10th and 11th bits of the lane change mode are set to 00 (instead of the default 01 or 10). 

A minimal working example is attached: 
[bugreport-lanechangemode.zip](https://github.com/eclipse/sumo/files/10967526/bugreport-lanechangemode.zip)

**SUMO-version:** 1.15

**operating system:** Windows 10

**Comment by m-kro** [comment]
I cannot reproduce this with the development version. For me both lane changes from the minimal working example are executed without calling `traci.vehicle.setLaneChangeMode`.

**Comment by mpfist-no** [comment]
You are right, my bad. I did some minor (or apparently not so minor) changes without testing again.

The behavior depends on the lane change duration parameter. Please see this example: 
[bugreport-lanechangemode.zip](https://github.com/eclipse/sumo/files/10968454/bugreport-lanechangemode.zip)


Setting the lane change duration to 10s allows the lane change, whereas setting it to 3s prevents it. If the lane change mode is set to do no automatic sublane changes, it works with both ways.

**Comment by m-kro** [comment]
Now I can reproduce the problem. It is related to the time the lane change is requested as well as to the lane change duration.

**Comment by m-kro** [comment]
@namdre Can you please help here? The second lane change (to the left) is rejected because the decision.latDist value is too low for a change - the evaluated direction is 0 = remain on the same lane...

https://github.com/eclipse/sumo/blob/b3ff74ceb4466d9892fbbd15a05695c85132439a/src/microsim/MSLaneChangerSublane.cpp#L241
